pkgs/development/python-modules/numba/default.nix

```diff
@@ -77,6 +78,11 @@ in buildPythonPackage rec {
       cuda_toolkit_lib_path = cudatoolkit.lib;
     })
   ];
+  # with untagged version we need to specify the correct version ourselves
+
+  postPatch = ''
+    substituteInPlace setup.py --replace "version=versioneer.get_version()" "version='0.57.1'"
```

Comment:
```suggestion
    substituteInPlace setup.py --replace "version=versioneer.get_version()" "version='0.57.1'"
    substituteInPlace numba/_version.py \
      --replace 'git_refnames = ""' 'git_refnames = "0.57.1"'
```

Reply:
This file does not exists here: https://github.com/numba/numba

Reply:
Honestly I think the structure of this automatically generated file is more likely to change than the fact that there is a version=versioneer.get_version() in this one.

Reply:
The proposed patch fixes python3Packages.awkward and such

Reply:
I misread your patch actually. I thought it was only patching `numba/_version.py`

Reply:
Oh, yes. I was contemplating proposing that, but in the end also decided that patching both files is more bulletproof.